.setup/install_submitty/get_globals.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+set -ve
+
+# We assume a relative path from this repository to the installation
+# directory and configuration directory.
+SETUP_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )/.."
```

Comment:
Sorry to keep harping on this variable (I know you didn't add it), but I don't think we need a special variable name for this.  It is just SUBMITTY_INSTALLATION_DIR + ".setup".

Can't we just define here:
SUBMITTY_INSTALLATION_DIR = "$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )/../.."
(adding an extra "/..")

And then eliminate SETUP_DIR everywhere.

Reply:
Didn't think of that, yeah that works. I'll do that now.